portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="map.js"></script>
+    <meta name="viewport" content="initial-scale=1.0, user-scalable=no">
+    <meta charset="utf-8">
+    <style>
+      /* Always set the map height explicitly to define the size of the div
+       * element that contains the map. */
+      #map {
+        height: 40%;
+        width: 40%;
+      }
+      /* Optional: Makes the sample page fill the window. */
+      html, body {
+        height: 100%;
+        margin: 0;
+        padding: 20px;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map"></div>
+    <script>
+      // Note: This example requires that you consent to location sharing when
+      // prompted by your browser. If you see the error "The Geolocation service
+      // failed.", it means you probably did not give permission for the browser to
+      // locate you.
```

Comment:
Did this come from a Maps API tutorial? If so, can you link me to it? I'm interested

Reply:
Yep! I think I lost the link that was meant to be in the comments originally do to an undo. The link is: https://developers.google.com/maps/documentation/javascript/geolocation

Linking it in the next commit as well, as a comment.

Reply:
map2.html does have default coordinates set to New York (arbitrary location), so it will load if no personalized coordinates are set from map.html. No other pages have geolocation set up. 
An alternative to the default coordinates is to have an error catch set up in the future once we move past the prototype phase, but I haven't looked into it yet.